src/playercards/spawnbag/BondedBag.ttslua

```diff
@@ -0,0 +1,63 @@
+require("playercards/spawnbag/SpawnBag")
+
+SPAWN_SPEC = {
+  name = "BondedCards",
+  cards = {
+    "05314", -- Soothing Melody
+    "06277", -- Wish Eater
+    "06019", -- Bloodlust
+    "06022", -- Pendant of the Queen
+    "05317", -- Blood-rite
+    "06113", -- Essence of the Dream
+    "06028", -- Stars Are Right
+    "06025", -- Guardian of the Crystallizer
+    "06283", -- Unbound Beast
+    "06032", -- Zeal
+    "06031", -- Hope
+    "06033", -- Augur
+    "06331", -- Dream Parasite
+    "06015a", -- Dream-Gate
+  },
+  globalPos = { x = -33.88, y = 1.5, z = 85.61 },
+  rotation = { x = 0, y = 270, z = 0 },
+  spread = true,
+}
+
+function onLoad(savedData)
+  if (savedData ~= nil) then
+    local saveState = JSON.decode(savedData)
+    if (saveState.spawnBagState ~= nil) then
+      SpawnBag.loadFromSave(saveState.spawnBagState)
+    end
+  end
+  createActionButtons()
+end
+
+function onSave()
+  local saveState = {
+    spawnBagState = SpawnBag.getStateForSave(),
+  }
+  return JSON.encode(saveState)
```

Comment:
Is it intended that the savestate is a table with one entry instead of that entry directly?
Also I wouldn't declare a new local variable if it's only used once (and place the table directly into the "JSON.encode()")

Reply:
It is.  Wrapping it in a save object is redundant when there's one, but minimally so and makes it much easier to see/change if more get added.

The extra local variable is a stylistic preference - I find embedded table declarations hard to read and format well, so generally avoid them.